### PR TITLE
Add KeePassRPC version 1.8.0 (plugin for KeePass 2.x)

### PR DESCRIPTION
**Pull request by ebugusey**
RPC plugin for [Kee](https://addons.mozilla.org/firefox/addon/keefox/) browser extensions.

### CODE REVIEW

File: bucket/keepass-plugin-keepassrpc.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "url": "https://github.com/kee-org/keepassrpc/releases/download/v1.8.0/KeePassRPC.plgx",
+    "hash": "8d9d5e390fc4a3b8d6d8f24dd26a712dc032c4ff49708c8ec32c95a2e27594b5",
+    "depends": "extras/keepass",
+    "checkver": {
+        "github": "https://github.com/kee-org/keepassrpc"
+    },
+    "autoupdate": {
+        "url": "https://github.com/kee-org/keepassrpc/releases/download/v$version/KeePassRPC.plgx"
+    },
+    "post_install": "Copy-Item \"$dir\\KeePassRPC.plgx\" \"$(appdir keepass)\\current\\Plugins\""
```
Comment (ebugusey) [inline bucket/keepass-plugin-keepassrpc.json]: I made this by example in [notepadplusplus-pm](https://github.com/lukesampson/scoop-extras/blob/c064d4d5d9d0773fdafb46ef831165d5bc00163e/bucket/notepadplusplus-pm.json). But I think it should be in `install` section, should I move it there?

Should I also make an uninstall section?

File: bucket/keepass-plugin-keepassrpc.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+    "homepage": "https://www.kee.pm/",
+    "description": "Kee is a Firefox and Chrome add-on for linking browsers to KeePass, using the KeePassRPC KeePass plugin contained within this repository.",
+    "license": "GPL-2.0-or-later",
```
Comment (ebugusey) [inline bucket/keepass-plugin-keepassrpc.json]: Also not sure about the license. It's certainly is [GPL2.0 ](https://github.com/kee-org/keepassrpc/blob/v1.8.0/LICENSE), but I don't if it's "2.0 and later" or "2.0 only".

**Comment by ebugusey** [comment]
@rasa  @r15ch13 some input, please?

File: bucket/keepass-plugin-keepassrpc.json (hunk 2 above)
Comment (rasa) [reply bucket/keepass-plugin-keepassrpc.json]: -or-later is fine.

File: bucket/keepass-plugin-keepassrpc.json (hunk 1 above)
Comment (rasa) [reply bucket/keepass-plugin-keepassrpc.json]: Yes, the user will expect uninstall to remove the .plgx file from the Plugins folder.

**Pull request merged**